lvsfunc/comparison.py

```diff
@@ -172,12 +170,12 @@ def tvbd_diff(tv: vs.VideoNode, bd: vs.VideoNode,
 
     if thr <= 1:
         diff = core.std.PlaneStats(tv, bd)
-        frames = [i for i,f in enumerate(diff.frames()) if f.props["PlaneStatsDiff"] > thr]
+        frames = [i for i, f in enumerate(diff.frames()) if f.props["PlaneStatsDiff"] > thr]
     else:
         diff = core.std.MakeDiff(tv, bd).std.PlaneStats()
-        frames = [i for i,f in enumerate(diff.frames()) if f.props["PlaneStatsMin"] <= thr or f.props["PlaneStatsMax"] >= 255 - thr]
+        frames = [i for i, f in enumerate(diff.frames()) if f.props["PlaneStatsMin"] <= thr or f.props["PlaneStatsMax"] >= 255 - thr]
 
-    if frames == []:
+    if not frames:
```

Comment:
I'm fairly sure this broke it back when I had it written like this.

Reply:
it should probably just be `if len(frames) == 0`

Reply:
pep8 says to keep it as `if not frames:`